wurlitzer.py

```diff
@@ -176,9 +176,13 @@ def forwarder():
             """Forward bytes on a pipe to stream messages"""
             draining = False
             flush_interval = 0
+            poller = select.poll()
+            for pipe_ in pipes: poller.register(pipe_)
```

Comment:
let's register these only for POLLIN so we only get readers, not writers:

```suggestion
            for pipe_ in pipes:
                poller.register(pipe_, select.POLLIN)
```